packages/components/src/components/search/SearchResultCard.tsx

```diff
@@ -63,7 +63,10 @@ export class SearchResultCard extends React.Component<SearchResultProps, any> {
         let altText;
         if (data) {
             if (data.getIn(['dataClass', 'name'])) {
-                iconSrc = data.getIn(['dataClass', 'name']).toLowerCase();
+                if ('sources' === data.getIn(['dataClass', 'category']))
+                    iconSrc = 'sources';
```

Comment:
We might want to do something more general here in the future, particularly since sources is SM-specific, but it seems ok for now.